src/router.ts

```diff
@@ -120,7 +120,17 @@ export class OneSchemaRouter<
         }
         return res.data;
       },
-      implementation,
+      async (ctx) => {
+        const result = await implementation(ctx);
+        const res = endpoint.response.safeParse(result);
+        if (!res.success) {
+          return ctx.throw(
+            500,
+            `A response value from endpoint '${route}' did not conform to the response schema.`,
```

Comment:
Is it worth exposing the exact error to the client? Maybe we should log and return internal server error?

Reply:
I agree that it might be nice to log the validation error to help with debugging.

Reply:
@david-wb Definitely. The good news: all errors thrown in `LifeOmicApp` instances are logged: https://github.com/lifeomic/koa/blob/master/src/LifeOmicApp.ts#L76-L87

I think it might be nice to avoid tying the `LoggerInterface` to this package and adding a custom log. Currently, the package doesn't know about loggers. Feels like it might be nice to keep it that way.

Reply:
Maybe you could add the validation error message string to this new error message for additional context?

Reply:
@david-wb good idea. on it.